bundles/org.openhab.binding.solarmax/src/main/resources/OH-INF/i18n/solarmax_it.properties

```diff
@@ -14,6 +14,8 @@ thing-type.config.solarmax.inverter.host.label = Host
 thing-type.config.solarmax.inverter.host.description = Nome host o indirizzo IP
 thing-type.config.solarmax.inverter.portNumber.label = Porta
 thing-type.config.solarmax.inverter.portNumber.description = Numero porta (valore predefinito 12345)
+thing-type.config.solarmax.inverter.deviceAddress.label = Indirizzo dispositivo
+thing-type.config.solarmax.inverter.deviceAddress.description = Indirizzo dispositivo (defaults to 1)
```

Comment:
This needs to be provided through Crowdin. For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
Deleted